src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/DefaultJavaScriptEncoder.cs

```diff
@@ -8,7 +8,7 @@
 using System.Text.Internal;
 using System.Text.Unicode;
 
-#if NETCOREAPP
+#if BUILDING_INBOX_LIBRARY
```

Comment:
You could do
#if !NETSTANDARD
...
#endif

And then that means anything but any version of netstandard.

Reply:
BUILDING_INBOX_LIBRARY is incorrect.  You want this for the package build of Netcoreapp.  I would recommend `#if netcoreapp` if that works in both master and release.  I know that works in master: versionless define indicates any, as it does with the SDK.  I think we still had the differing behavior in release, so a dedicated define may be appropriate.  How about `USE_INTRINSICS` or something like that in release?

Reply:
Yes, the SDK constants do work in the release/3.x branches as well.

Reply:
> I would recommend `#if netcoreapp` if that works in both master and release.

Any difference between lower-case and upper-case? I left it as upper-case since that's what is being used across the repo.

Reply:
Lower-case is what we used to define but we removed that some weeks ago, upper-case are the ones the SDK define based on the TFM.